### PR TITLE
Added | Event | Objective-See

### PR DESCRIPTION
**Pull request by irshadpc**
Added | Event | Objective-See

### CODE REVIEW

**Comment by irshadpc** [comment]
@pedrommcarrasco ,I don't know how to add logo. Please guide me through this.

File: Events/30-Objective-See.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Objective by the Sea",
+    "url": "https://objectivebythesea.com/v3/",
+    "logo": "",
+    "startDate": "2020-03-12T12:00:00+0000",
+    "endDate": "2020-03-13T12:00:00+0000",
+    "country": "USA",
```
Comment (pedrommcarrasco) [inline Events/30-Objective-See.json]: Replace with `United States of America` please

**Comment by pedrommcarrasco** [comment]
> @pedrommcarrasco ,I don't know how to add logo. Please guide me through this.

There you go https://firebasestorage.googleapis.com/v0/b/pedrommcarrasco-cocoahub.appspot.com/o/events%2FobjectiveSee.png?alt=media&token=5c6caf16-d4f8-4ef7-b083-a0fe84c1cc3a 

Thanks for adding this one, never heard of it!

File: Events/30-Objective-See.json (hunk 1 above)
Comment (irshadpc) [reply Events/30-Objective-See.json]: Fixed.

**Pull request merged**